base/applications/network/ping/ping.c

```diff
@@ -419,20 +420,26 @@ void
 Ping(void)
 {
     PVOID ReplyBuffer;
-    PVOID SendBuffer = NULL;
+    PUCHAR SendBuffer = NULL;
     DWORD ReplySize = 0;
     DWORD Status;
 
     if (RequestSize != 0)
     {
-        SendBuffer = malloc(RequestSize);
+        SendBuffer = (PUCHAR)malloc(RequestSize);
         if (SendBuffer == NULL)
         {
             ConResPrintf(StdErr, IDS_NO_RESOURCES);
             exit(1);
         }
 
-        ZeroMemory(SendBuffer, RequestSize);
+        // Windows ping utility fills the optional data field
+        // with ASCII characters from 'a' to 'w' wrapping back around
+        // until SendBuffer is full
+        for(int i = 0; i  < RequestSize; i++)
+        {
+            SendBuffer[i] = (UCHAR)(97 + (i % 23));
+        }
```

Comment:
```suggestion
        for (int i = 0; i < RequestSize; i++)
            SendBuffer[i] = (UCHAR)(97 + (i % 23));
```
And can't magic value `97` be replaced by something more explicit? And maybe `23` too.

Reply:
I added two constants ALPHABET_START 97 and ALPHABET_LENGTH 23